src/models.rs

```diff
@@ -77,6 +77,8 @@ pub struct Upload {
     pub forc_version: String,
     pub abi_ipfs_hash: Option<String>,
     pub bytecode_identifier: Option<String>,
+    pub readme: Option<String>,
+    pub forc_manifest: Option<String>,
```

Comment:
wondering shouldnt a manifest always be required?
Are there scenarios where this may not be the case?

Reply:
Good catch, this should not be optional 👍